docs/contributing/contributing-rules.md

```diff
@@ -19,13 +26,17 @@ There are two ways in which you can contribute to the Semgrep rules repository:
 
 ### Contributing through Semgrep App (recommended)
 
-To contribute to the Semgrep rules repository through Semgrep App, follow these steps:
+Publish rules in the open-source Semgrep Registry to share them with the Semgrep community. To contribute and publish rules to the Semgrep Registry, follow these steps:
+
 1. Go to [Semgrep App Editor](https://semgrep.dev/orgs/-/editor).
 2. Click **Create New Rule**.
-3. Make one of the following steps:
-    - Create a new rule and test code, and then click **Save**. Note: The test file must contain at least one true positive and one true negative test case to be approved. See the [Tests](#tests) section of this document for more information.
-    - Select a rule from a category in **Semgrep Registry**. Modify the rule or test code, click **Save**, and then **Fork**.
-4. Click **Share**.
+3. Choose one of the following steps:
```

Comment:
+1 choose. yay! May I suggest you also say "choose one of the following options" (and go back to a bulleted list) because steps are often numbered, and followed in a sequence, whereas options don't have an order and you choose from them. :-)

Reply:
I would normally go with a lettered list after this step, but I have not figured out CSS tricks on how to do it in our Docusaurus.

Reply:
So I will go with a bullet point list, and sentence fragment can work in this case also (`3. Choose one of the following:`).